src/scopes/activityReport/feiRootCause.js

```diff
@@ -0,0 +1,21 @@
+import { filterAssociation } from './utils';
+
+const feiResponseSql = `
+WITH unnested_responses AS (
+    SELECT "activityReportGoalId", unnest("response") AS res
+    FROM "ActivityReportGoalFieldResponses"
+)
+SELECT
+  DISTINCT "ActivityReportGoals"."activityReportId"
+FROM "ActivityReportGoals" "ActivityReportGoals"
+INNER JOIN unnested_responses arr
+    ON arr."activityReportGoalId" = "ActivityReportGoals"."id"
+WHERE arr."res"`;
+
+export function withFeiRootCause(responses) {
+  return filterAssociation(feiResponseSql, responses, false);
+}
+
+export function withoutFeiRootCause(responses) {
```

Comment:
The way the query is written, the negative match will only match those with something populated that is not what was specified. It will not match those with nothing populated.

Reply:
This appears to be behavior in other filters like author, collaborators, grant numbers, program specialist, recipient, report text, etc....